packages/reactive_ble_mobile/android/src/main/kotlin/com/signify/hue/flutterreactiveble/PluginController.kt

```diff
@@ -267,4 +268,18 @@ class PluginController {
                 })
                 .discard()
     }
+
+    private fun readRssi(call: MethodCall, result: Result) {
+        val args = pb.ClearGattCacheRequest.parseFrom(call.arguments as ByteArray)
```

Comment:
I don't think `ClearGattCacheRequest` should be used for subscribing to RSSI updates.